OpenJDK 8.0 doesn't load timeshape

# OpenJDK 8.0 doesn't load timeshape

- Code compiles.
- But throws a Runtime error:

```bash
net/iakovlev/timeshape/TimeZoneEngine has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```

## Which Timeshape version are you using?

```gradle
dependencies {
    implementation(project(':common'))
    implementation(project(':domain'))

    implementation(group: 'org.apache.commons', name: 'commons-lang3', version: '[3.7,4)')
    implementation(group: 'org.apache.commons', name: 'commons-math3', version: '[3.6.1,4)')

    implementation(group: 'net.time4j', name: 'time4j-base', version: '[5.9,6)')
    implementation(group: 'net.time4j', name: 'time4j-tzdata', version: '[5.0-2023b,6)')

    implementation(group: 'net.iakovlev', name: 'timeshape', version: '[2022g.17,)')

}
```

## Which Java version are you using?

```bash
$ java -version
openjdk version "1.8.0_362"
OpenJDK Runtime Environment (build 1.8.0_362-8u372-ga~us1-0ubuntu1~23.04-b09)
OpenJDK 64-Bit Server VM (build 25.362-b09, mixed mode)
```

## If you upgrade to the latest Java minor version, does it still fail?

Previous working version:

``` gradle
    // group: 'net.iakovlev', name: 'timeshape', version: '[2020d.11,)'
```

## Additional information
E.g. if this used to work in some previous Timeshape version or any other significant facts.

```java

    public static void initTimeZoneSouthAmerica() {

        // Shapes from South America Rectangle
        timeZoneEngine = TimeZoneEngine.initialize(-56.0, -83.0, 13.0, -32.0, true);
    }
```


```text
at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.UnsupportedClassVersionError: net/iakovlev/timeshape/TimeZoneEngine has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:473)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	at com.movida.base.TimeZoneCommon.initTimeZoneSouthAmerica(TimeZoneCommon.java:110)
TimeZoneCommon.java:110
	at com.movida.base.TimeZoneCommon.initTimeZone(TimeZoneCommon.java:85)
TimeZoneCommon.java:85
	at com.movida.base.TimeZoneCommon.queryZoneNameFrom(TimeZoneCommon.java:40)
TimeZoneCommon.java:40
	at com.movida.base.TimeZoneCommon.getLocalDateTimeAt(TimeZoneCommon.java:30)
TimeZoneCommon.java:30
```

--- a/build.sbt
+++ b/build.sbt
@@ -3,6 +3,9 @@ import _root_.io.circe.parser._
 
 val dataVersion = "2022g"
 val softwareVersion = "18-SNAPSHOT"
+
+scalacOptions += "-target:jvm-1.8"
+
 val `commons-compress` = Seq(
   "org.apache.commons" % "commons-compress" % "1.22",
   "com.github.luben" % "zstd-jni" % "1.5.2-5"

